qa-tests-backend/src/test/groovy/NetworkBaselineTest.groovy

```diff
@@ -101,11 +103,10 @@ class NetworkBaselineTest extends BaseSpecification {
     }
 
     def validateBaseline(NetworkBaselineOuterClass.NetworkBaseline baseline, long beforeCreate,
-                         long justAfterCreate, List<Tuple2<String, Boolean>> expectedPeers) {
+                         long justAfterCreate, List<Tuple2<String, Boolean>> expectedPeers, List<String> explicitMissingPeers) {
```

Comment:
Fred, this is okay the way it is. My original comment was more about making this naming more explicit, for example:

```suggestion
                         long justAfterCreate, List<Tuple2<String, Boolean>> mustFindPeers, List<String> mustNotFindPeers) {
```

or similar

Reply:
Done!